src/NativeScript/inspector/GlobalObjectInspectorController.cpp

```diff
@@ -125,7 +125,19 @@ GlobalObjectInspectorController::~GlobalObjectInspectorController() {
 }
 
 void GlobalObjectInspectorController::registerDomainDispatcher(WTF::String domainIdentifier, JSC::JSCell* constructorFunction) {
-    std::unique_ptr<DomainInspectorAgent> domainInspectorAgent = std::make_unique<DomainInspectorAgent>(m_globalObject, domainIdentifier, constructorFunction);
+    AgentContext baseContext = {
```

Comment:
Yeah, I've found a stackoverflow answer that said the same thing, but since it's not cool to modify the source I guess I am stuck with creating a context on every domain register